subjects/own-and-return/README.md

```diff
@@ -0,0 +1,55 @@
+## own_and_return
+
+### Instructions
+
+Create a struct `Film` that has one field `name` of type `String`.
+
+Create the following two functions:
+
+- Create a function `take_film_name` it will return the name and consume the film (you should not be able to reuse it after you passed it to the function).
+- Create a function `read_film_name` it will return the name without consuming the film (you can call the function multiple times with the same argument).
+
+### Expected functions
+
+```rust
+pub struct Film {
+    pub name: String,
+}
+
+pub fn read_film_name(/* to be implemented */) -> String {
+}
+
+pub fn take_film_name(/* to be implemented */) -> String {
+}
+
+
+```
+
+### Usage
+
+Here is a possible program to test your function :
```

Comment:
Good catch, I agree with you, I copy pasted this and it slipped through, I'll be more careful now. It would be nice to fix all the  subjects in the future when priorities allow it :slightly_smiling_face: